core/metacling/src/TClingClassInfo.cxx

```diff
@@ -115,7 +115,7 @@ TClingClassInfo::TClingClassInfo(cling::Interpreter *interp, const char *name, b
 TClingClassInfo::TClingClassInfo(cling::Interpreter *interp,
                                  const Type &tag)
    : TClingDeclInfo(nullptr), fInterp(interp), fFirstTime(true), fDescend(false), fIterAll(kTRUE),
-     fIsIter(false), fType(nullptr), fTitle(""), fOffsetCache(0)
```

Comment:
I thought clang only complained about fType, why removing also fTitle?

Reply:
That is just me noticing that for an std::string the notation `fTitle("")` has the same effect as calling the default constructor.